pycbc/inference/models/tools.py

```diff
@@ -363,7 +368,7 @@ def draw_times(self, snrs, size=None):
         starts = []
         ends = []
 
-        tmin, tmax = tcmin - dt, tcmax + dt
+        tmin, tmax = tcmin + dt - snrs[iref].delta_t, tcmax + dt + snrs[iref].delta_t
```

Comment:
Why did you change `tcmin - dt` to `tcmin + dt`? Is `snrs[iref].delta_t` the time resolution of the SNR time series for a detector?

Reply:
To make sure the boundary condition was satisfied.